### ISSUE TITLE
the crop_to_shape(data, shape) function not work right

### ISSUE THREAD

**Issue by LilianYe** [jakeret/tf_unet#89]
when (data.shape[1] - shape[1]) % 2 == 1, the return size is shape[1]+1 but not shape[1].

**Comment by jakeret** [comment]
Yes this has been an issue for a while. Could you send me a Pull Request for that?

**Comment by lixiangamy** [comment]
feed_dict={self.net.x: batch_x,
                                   self.net.y: util.crop_to_shape(batch_y, pred_shape),
                                   self.net.keep_prob: dropout})

why  batch_y.shape!=pred_shape?

**Comment by jakeret** [comment]
Due to the network architecture the output prediction are always smaller than the input labels
